dask/dataframe/tests/test_rolling.py

```diff
@@ -270,7 +270,7 @@ def test_time_rolling_constructor():
 @pytest.mark.parametrize("window", ["1S", "2S", "3S", pd.offsets.Second(5)])
 def test_time_rolling_methods(method, args, window, check_less_precise):
     if dd._compat.PANDAS_GT_110:
-        check_less_precise = {}
+        check_less_precise = {"atol": 1e-4, "rtol": 1e-2}
```

Comment:
```suggestion
                if check_less_precise:
                    check_less_precise = {"atol": 0.5e-3, "rtol": 0.5e-3}
                 else:
                     check_less_precise = {"atol": 0.5e-5, "rtol": 0.5e-5}
```

Sorry, I didn't think about this when updating. Looking at https://github.com/pandas-dev/pandas/blob/6ea8474f502a14a97fd1407e9a073228abe36c7e/pandas/_testing.py#L340, I think that setting these to the suggested values should be equivalent across versions (haven't tested though)..

Reply:
Ah that makes sense. My change did seem rather aggressive.